server/models/Collective.js

```diff
@@ -2619,14 +2619,25 @@ function defineModel() {
             tier.presets = null;
             tier.minimumAmount = null;
           }
-          if (tier.invoiceTemplate) {
-            tier.data = { ...tier.data, invoiceTemplate: tier.invoiceTemplate };
-          }
           if (tier.id) {
-            return models.Tier.update(tier, { where: { id: tier.id, CollectiveId: this.id } });
+            models.Tier.findOne({ where: { id: tier.id, CollectiveId: this.id } }).then(t => {
+              if (tier.invoiceTemplate) {
+                tier.data = { ...t.data, invoiceTemplate: tier.invoiceTemplate };
+              }
+              if (tier.singleTicket !== undefined) {
+                tier.data = { ...t.data, singleTicket: tier.singleTicket };
+              }
+              return models.Tier.update(tier, { where: { id: tier.id, CollectiveId: this.id } });
+            });
```

Comment:
Here also, why reload the tier?

Reply:
Apparently no as `tier.data` is always undefined here. I've missed this in a previous PR as well, for the `invoiceTemplate` condition. 🤔

Reply:
@Betree : And the reason this is always undefined is that we deliberately return an empty object as per, 

https://github.com/opencollective/opencollective-api/blob/b82593b50de6e78f05b7331137ebf4c7dec441b7/server/graphql/v1/CollectiveInterface.js#L1140-L1146. 

So I think here the reloading is justified. 🤔

Reply:
We already load the tier above with `this.getTiers().then(oldTiers =>`, it's just a matter of re-arranging the code and passing the right variable. It will make the code clearer and more performant.

Reply:
This part is no longer needed as with the new Tier page we are not using this at all. I've removed these changes.